packages/frontend/web/server/htmlTemplate.ts

```diff
@@ -2,6 +2,7 @@ import resetCSS from /* preval */ '@frontend/lib/reset-css';
 import { getFontsCss } from '@frontend/lib/fonts-css';
 import { getStatic } from '@frontend/lib/assets';
 import { prepareCmpString } from '@frontend/web/browser/prepareCmp';
+import escape from 'lodash.escape';
```

Comment:
This overrides `escape` used on line 56 and matches AMP implementation of the same.